src/test/java/com/uid2/admin/vertx/SharingServiceTest.java

```diff
@@ -357,9 +381,11 @@ void KeysetSetNoSiteIdOrKeysetId(Vertx vertx, VertxTestContext testContext) {
     }
 
     @Test
-    void KeysetSetBothSiteIdAndKeysetId(Vertx vertx, VertxTestContext testContext) {
+    void KeysetSet(Vertx vertx, VertxTestContext testContext) {
```

Comment:
should this be named something more descriptive - e.g. KeysetUpdateAllowSitesSuccessful?

Reply:
changing this to KeysetCanUpdateAllowedSitesAndName